### ISSUE TITLE
Why does Minimal API Scaffold generated `PUT` endpoint not update the model properties?

### ISSUE THREAD

**Issue by brunolins16** [dotnet/Scaffolding#1964]
The code generated by the Minimal API Scaffolder includes:
- Get all items
- Get item by key
- Update existing item
- Add a new item
- Delete an item

All of them contain the minimal required code to do what is expected, except the **Update existing item** that misses the step to update all the model properties with the new provided value and instead include the comment `//update model properties here`

``` C#
routes.MapPut("/api/SampleModel/{id}", async (int Id, SampleModel sampleModel, SampleModelContext db) =>
{
    var foundModel = await db.Samples.FindAsync(Id);

    if (foundModel is null)
    {
        return Results.NotFound();
    }
    //update model properties here

    await db.SaveChangesAsync();

    return Results.NoContent();
})
```

Is that the expected behavior? What is the reason behind this decision?

**Comment by DamianEdwards** [comment]
I think this is fixed now given #2038?

**Comment by deepchoudhery** [comment]
@DamianEdwards @brunolins16 yep, fixed now in the latest 7.0.0 package for .NET 7 but not .NET 6. Should we service this change back @DamianEdwards ?

**Comment by DamianEdwards** [comment]
Yes I think we should.

**Comment by deepchoudhery** [comment]
Cool, I'll close it once the servicing packages go out next week.
